### PR TITLE
Bump minimum Elixir version to 1.11

### PR DESCRIPTION
**Pull request by wkirschbaum**
Since Elixir 1.15 is out, 1.7-1.10 is not supported anymore. This also
allows us to get rid of deprecation warnings.

https://hexdocs.pm/elixir/1.15.0/compatibility-and-deprecations.html

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,17 +2,30 @@ on: push
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (wkirschbaum) [inline .github/workflows/ci.yml:9]: We need an older version of ubuntu to test against otp 23 and 24

**Comment by patrickdet** [comment]
Hi @wkirschbaum thank you for this PR! The CI still says: "Workflow runs completed with no jobs". Once the CI works I'll gladly merge this and finally release a new version.

**Comment by wkirschbaum** [comment]
@patrickdet is that not a setting on this repo? It ran successfully on our account: https://github.com/floatpays/fuzzy_compare/actions/runs/6559958531

( maybe a trigger rule, let me check )

**Comment by wkirschbaum** [comment]
There we go
image

**Comment by wkirschbaum** [comment]
@patrickdet Is there anything still required from this PR?

**Pull request merged**